paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt

```diff
@@ -312,6 +313,40 @@ internal object ImageUtils {
     }
   }
 
+  fun smallestDiffRect(firstImage: BufferedImage, secondImage: BufferedImage): Rectangle? {
+    var rect: Rectangle? = null
+    val width = max(firstImage.width, secondImage.width)
+    val height = max(firstImage.height, secondImage.height)
+
+    for (y in 0 until height) {
+      for (x in 0 until width) {
+        val firstRgb = if (x < firstImage.width && y < firstImage.height) firstImage.getRGB(x, y) else 0
```

Comment:
`0` is a real value here; black. Is that okay?

Reply:
Yes it's ok, if they both end up being black then nones the wiser as the pixel "hasn't" changed between the images so we don't need this specific pixel, also if the images are different sized then we likely are failing and will be generating the delta gif which would cover this case

Reply:
Even if firstImage is less wide than secondImage, and secondImage has black/non-black pixels on the wider part?

Reply:
Good call. That would leave the resulting frame's pixel transparent instead of black if first frame didn't contain the pixel and second contained black.